api/src/main/java/ca/bc/gov/educ/api/batchgraduation/config/BatchJobConfig.java

```diff
@@ -157,6 +185,16 @@ public Step archiveSchoolReportsJobStep(JobRepository jobRepository, PlatformTra
                 .build();
     }
 
+    @Bean
+    public Step deleteStudentReportsJobStep(JobRepository jobRepository, PlatformTransactionManager transactionManager, EducGradBatchGraduationApiConstants constant) {
+        return new StepBuilder("deleteStudentReportsJobStep", jobRepository)
+                .<List<UUID>, List<UUID>>chunk(constant.getTransactionChunkSize(), transactionManager)
+                .processor(deleteStudentReportsProcessor())
```

Comment:
The flow here is correct?    Readability & Consistency wise, reader should be first and then processor.

Reply:
Fixed, but really it doesn't matter. It is just the builder. The engine will call a reader first any way.